submissions/mustbefail/friends-app/src/js/app.js

```diff
@@ -32,10 +32,11 @@ const getUsers = async (url) => {
     } else {
       const json = await response.json();
       const users = await json.results;
-      return users;
+      state.users = [...users];
```

Comment:
Nope, previous idea was right. `getUsers` should only fetch users and not write them or process them.